result/lldpneighbors.yaml

```diff
@@ -0,0 +1,155 @@
+components:
+  schemas:
+    LldpNeighbors.State.Request:
+      description: >-
+        The request to retrieve LLDP per interface neighbor information.
+      type: object
+      properties:
+        lldp_names:
+          description: >-
+            The names of LLDP interfaces for which neighbor information will be
+            retrieved.If no names are specified then the results will
+            contain neighbor information for all configured LLDP interfaces.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.Lldp/properties/name"
+        id_filters:
+          description: >-
+            Specify which neighbors to return. If the list is empty or missing
+            then all neighbors will be returned.
```

Comment:
"Specify which neighbors to return." --> "Specify the neighbors for which information will be returned"

Reply:
updated